apps/yggdrasil/lib/yggdrasil/room.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Yggdrasil.Room do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Yggdrasil.Room
+
+  schema "rooms" do
+    field :title, :string
+    field :description, :string
+    field :is_starting, :boolean, default: false
+    timestamps
+  end
+
+  def create_changeset(params) do
+    %Room{ }
```

Comment:
Ah, yeah we used `create_changeset` in the `User` module and that was because it was a bit of an odd case where the password hash was being generated which would not want not want when updating a user etc.

In your case I think `changeset` would be fine. I think I have used `changeset` in all the others like the `Character` module and any of the newer ones added.